library/bignum_mod_raw.h

```diff
@@ -170,6 +170,41 @@ void mbedtls_mpi_mod_raw_sub( mbedtls_mpi_uint *X,
                               const mbedtls_mpi_uint *B,
                               const mbedtls_mpi_mod_modulus *N );
 
+/** \brief  Multiply two MPIs, returning the residue modulo the specified
+ *          modulus.
+ *
+ * \note Currently handles the case when `m->int_rep` is
```

Comment:
```suggestion
 * \note Currently handles the case when `N->int_rep` is
```

Reply:
Should it be "only handles the case..."?